exporter/datadogexporter/config.go

```diff
@@ -242,6 +242,19 @@ type TracesConfig struct {
 	flushInterval float64
 }
 
+// LogsConfig defines logs exporter specific configuration
+type LogsConfig struct {
+	// TCPAddr.Endpoint is the host of the Datadog intake server to send logs to.
+	// If unset, the value is obtained from the Site.
+	confignet.TCPAddr `mapstructure:",squash"`
+
+	// SendLogRecordBody when enabled would send string representation of body field of log record as message to Datadog
+	// By default this is enabled ,when we parse the body string into attributes we may want to disable
+	// Set this to true, if we are unable to parse body or we want use datadog backend for parsing
+	// It is essential for Log Trace correlation to work, that we parse the body before reaching exporter.
```

Comment:
It's not very clear what is being said here. Generally, boolean types are documented using the word "reports" in Go. I'd recommend:
```go
// SendLogRecordBody reports whether the log record's body should be sent as a message to Datadog 
// (instead of an attribute). It defaults to true.
// You may want to disable it if you wish for the body to be sent as an attribute instead.
```
The rest of the test is unclear. Who should set this true? Who is the "we" that is unable to parse the body? When does the Datadog backend do the parsing? Why is this essential for log correlation?

If I can't understand it as a Datadog employee, the chances of a user understanding it are quite low. Perhaps we should remove the last part that is unclear and just leave what I suggested which is clear enough. Or maybe simply adding a sentence explaining clearly when you'd ever want to turn it off.

You can also consider renaming it as:
```go
BodyAsAttribute bool `mapstructure:"body_as_attribute"`
```
It's much easier to understand, and it defaults to `false` (because Go booleans default to false), which is the equivalent of what you have now, basically the body is NOT sent as an attribute. I would not re-use the word "log" in a section that already contains the word "logs" if that makes sense. The context of the config and discussion is already "logs".